azblob/url_block_blob.go

```diff
@@ -154,6 +154,22 @@ func (bb BlockBlobURL) CopyFromURL(ctx context.Context, source url.URL, metadata
 		nil, // Blob ifTags
 		dstLeaseID, nil, srcContentMD5,
 		blobTagsString, // Blob tags
-		nil,            // seal Blob
 	)
 }
+
+// PutBlobFromURL synchronously creates a new Block Blob with data from the source URL up to a max length of 256MB.
+// For more information, see https://docs.microsoft.com/en-us/rest/api/storageservices/put-blob-from-url.
+func (bb BlockBlobURL) PutBlobFromURL(ctx context.Context, h BlobHTTPHeaders, source url.URL, metadata Metadata, srcac ModifiedAccessConditions, dstac BlobAccessConditions, srcContentMD5 []byte, dstContentMD5 []byte, tier AccessTierType, blobTagsMap BlobTagsMap, cpk ClientProvidedKeyOptions) (*BlockBlobPutBlobFromURLResponse, error) {
+
+	srcIfModifiedSince, srcIfUnmodifiedSince, srcIfMatchETag, srcIfNoneMatchETag := srcac.pointers()
+	dstIfModifiedSince, dstIfUnmodifiedSince, dstIfMatchETag, dstIfNoneMatchETag := dstac.ModifiedAccessConditions.pointers()
+	dstLeaseID := dstac.LeaseAccessConditions.pointers()
+	blobTagsString := SerializeBlobTagsHeader(blobTagsMap)
```

Comment:
Code cleanliness: If the order of .pointers() matches a subsection of the function, you can place .pointers() in there and move along.

Reply:
Tried doing this but Golang doesn't seem to recognize that the '.pointers()' returns the desired subsection and thinks it is only filling one parameter.